docs/installation.md

```diff
@@ -229,3 +229,93 @@ keycloak                         <none>   keycloak.127.0.0.1.nip.io   localhost
 
 You can use impress on https://impress.127.0.0.1.nip.io. The provisionning user in keycloak is impress/impress.
 
+## Options
```

Comment:
This is a very good idea I plan to make, so thanks a lot for your contribution.

Is it possible for you to create a new file in `docs` and put this in a file `env.md` and we will dedicate this file to environment variable ?